src/include/network/lan/address.rb

```diff
@@ -1593,14 +1546,72 @@ def AddressDialog
 
   private
 
-    def get_selected_bridges
+    # Checks if the current LanItem is a bridge or an enslaved interface part
+    # of a bridge. In case that some configuration is not correct then adapt it
+    # if the user accepts.
+    def check_bridge_ports_config
+      return false unless LanItems.type == "br"
```

Comment:
Just thinking ...
... using ```NetworkInterfaces.GetType``` makes no sense when creating new bridge until new configuration is commited into ```NetworkInterfaces```
... so you can just move the type check out of this method.